client/state/ui/reducer.js

```diff
@@ -25,6 +26,31 @@ export function selectedSite( state = null, action ) {
 	return state;
 }
 
+export function section( state = false, action ) {
+	if ( action.type === SET_SECTION && action.section !== undefined ) {
+		state = action.section;
+	}
+	return state;
+}
+
+export function noSidebar( state = false, action ) {
```

Comment:
Given we are moving things around, I think we should rename this to be the opposite `sidebar: true`.